R/derive_params_growth_height.R

```diff
@@ -227,11 +227,14 @@ derive_params_growth_height <- function(dataset,
     cli_abort("One of `set_values_to_sds`/`set_values_to_pctl` has to be specified.")
   }
 
+  bins <- get_bins(meta_criteria, param = "HEIGHT_LENGTH")
+
   # create a unified join naming convention, hard to figure out in by argument
   dataset <- dataset %>%
     mutate(
       sex_join := {{ sex }},
-      heightu_join := {{ height_unit }}
+      heightu_join := {{ height_unit }},
+      ht_bins := map({{ height }}, ~ set_bins(.x, breaks = bins$breaks, labels = bins$labels))
```

Comment:
`ht_bins` should be defined as a global variable, I get the following message while running the checks "_derive_params_growth_height: no visible binding for global variable 'ht_bins'_ "